### ISSUE TITLE
Missing config of normalization layer - keras.applications.EfficientNetB0

### ISSUE THREAD

**Issue by ronigober** [keras-team/tf-keras#636]
**System information**.

- OS Platform and Distribution: Windows 10
- TensorFlow installed from: pip install
- TensorFlow version : 2.8.0
- Python version: 3.8.0
- GPU model and memory: 3060 Ti 8GB

**Describe the problem**.

When using [to_json](https://keras.io/api/models/model_saving_apis/#tojson-method) function for later reloading of tf.keras.application.EfficientNetB0, the layer configuration is missing.

**Describe the current behavior**.

Layer's json configuration is:
{"class_name": "Normalization", "config": {"name": "normalization_1", "trainable": true, "dtype": "float32", "axis": [3], "mean": null, "variance": null}, "name": "normalization_1", "inbound_nodes": [[["rescaling_1", 0, 0, {}]]]}

While in the normalization layer inside model.layers, "mean", "variance", "adapt_mean", "adapt_variance", "non_trainable_weights", "non_trainable_variables", "weights", "non_trainable_variables", are holding the well known mean and variance of imagenet dataset: [0.485, 0.456, 0.406], [0.229, 0.224, 0.225].

In addition, "count" weights of normalization layer are missing too.

**Describe the expected behavior**.

I expect that the right parameters will be exported in config() - self.mean & self.variance.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**.

- Do you want to contribute a PR? (yes/no): Yes, but not sure, where I need to make this change.

**Standalone code to reproduce the issue**.

from tensorflow.keras.applications import EfficientNetB0
model = tf.keras.applications.efficientnet.EfficientNetB0(weights='imagenet')
json = model.to_json()
